create_object.yml

```diff
@@ -20,9 +20,11 @@
 
   tasks:
 
-  - debug: msg="Project name {{ project_name }} - domain {{ domain_name }}"
+  - name: Print project informations
+    debug: msg="Project name {{ project_name }} - domain {{ domain_name }}"
 
-  - debug: msg="Object template {{ object_template}}"
+  - name: Print Object template
+    debug: msg="Object template {{ object_template}}"
```

Comment:
```yaml
debug: msg="Object template {{ object_template }}"
```
note the space missing around variable name (this was not introduced by your work but certainly mine :wink:)